### ISSUE TITLE
Cleanup Inactive Org Members

### ISSUE THREAD

**Issue by palnabarun** [kubernetes/org#3094]
Following is the list of [inactive members](https://github.com/kubernetes/community/blob/master/community-membership.md#inactive-members) who are slated to be removed from the Kubernetes organizations: 

**TBD** (pending on a data import from devstats)

### Tasks
- [ ] Get the data import
- [ ] Find members to remove and teams they belong to
- [ ] Ping and wait for people to let us know if they have been contributing to non-code areas
- [ ] Remove the inactive members

**Comment by palnabarun** [comment]
Previous Instances of cleanup: https://github.com/kubernetes/org/issues/2456, https://github.com/kubernetes/org/issues/2076

**Comment by mrbobbytables** [comment]
/assign
(just to keep it on my radar)

**Comment by mrbobbytables** [comment]
/retitle Cleanup Inactive Org Members -Q1 2022

**Comment by nikhita** [comment]
/unassign @nikhita @MadhavJivrajani @RinkiyaKeDad 

Let's re-assign when tasks are being picked up.

@palnabarun do you know how to import the required data from devstats? If yes, can you also help document this so either of us can pick this up in the future?

**Comment by Sea-n** [comment]
We still need to cleanup inactive org members.
/remove-lifecycle stale

It's being postponing, remove the time from issue title.
/retitle Cleanup Inactive Org Members

**Comment by nikhita** [comment]
/remove-lifecycle frozen

@palnabarun or @mrbobbytables we've added a lot of members in the last few months. Do y'all have bandwidth to cleanup inactive members soon?

**Comment by pacoxu** [comment]
/assign 
I could like to help once we need to do a cleanup.

**Comment by palnabarun** [comment]
PR #4291  introduces a command to audit the orgs. There are few TODOs to be done as followups after it is merged, most important being removing org members using the `korg` toolkit. I will take that up next week to ease the removal process.

**Comment by pacoxu** [comment]
It seems that we did not cleanup inactive org members for two years. 

Can I try with @palnabarun's tool in the PR to do a cleanup this month?

**Comment by pacoxu** [comment]
The removal list is at https://gist.github.com/pacoxu/21622b30d3511d8a9df8507f58926769. 

I would open a new issue  "Clean up inactive members - Sept 2023" to track the removal process if there is no question about it.

**Comment by palnabarun** [comment]
@pacoxu I will send comms about the removal next week after verifying the audit script. We need to give the community a heads-up before removing org members.

Also, I don't think we need to open a new issue. We can work out of this.

**Comment by palnabarun** [comment]
Following up on https://github.com/kubernetes/org/issues/4726

/close
